src/main/resources/templates/email/observation/plotReplaced/body.ftlh.mjml

```diff
@@ -8,19 +8,22 @@
 
         <mj-section padding="0px">
             <mj-column mj-class="body-wrapper">
-                <#if !hasPrimaryContact>
-                    <mj-text mj-class="text-body03">
-                        ${strings("notification.observation.monitoringPlotReplaced.email.body.1")}
-                    </mj-text>
-                </#if>
+                <mj-text mj-class="text-headline06-bold">
+                ${strings("notification.observation.monitoringPlotReplaced.email.body.1", organizationName)}
+                </mj-text>
+                <mj-text mj-class="text-body03">
+                  <!-- htmlmin:ignore --><#if !hasPrimaryContact><!-- htmlmin:ignore -->
```

Comment:
Got this to work by moving it inside the mjml tag, looked at device/unknownAutomation as a guide.
When the ignore tags were outside mjml, the freemarker tags were not being preserved (looked in build/resource/..templates/email/..plotReplaced/body.ftlh).

Reply:
That makes sense now that I see it. The MJML tool doesn't know anything about those comments, just treats them as content, so if they appeared where it didn't expect content, it would just treat them as if they were actual comments and ignore them.